### ISSUE TITLE
Graph trend window hint and improvements

### ISSUE THREAD

**Issue by kklausmeyer** [tnc-ca-geo/gde#107]
Add a hint to make it clear that the grey box is moveable.  Most people don't know they can move it, but they really like it when I demonstrate it.  

Also consider adding the year range to the top of the chart near the trend value.  This will help consultants quickly write up stats about their GDEs of interest.  E.g. "1999-2018 average annual change: -1.9".  I know this is too long but something like that.

**Comment by nathanielrindlaub** [comment]
I added start and end years to the bottom of the trend window to make it clear what years the trend window covered. I also tried a bunch of design ideas out but couldn't figure out an intuitive way to indicate that the chart was movable. My hope is that we can explain this in the "chart help" page and add a demo video. See [gde.codefornature.org/#/chart-help](https://gde.codefornature.org/#/chart-help) for an idea of what I'm thinking. 

Feel free to close out if that all works for you.
